examples/js/ColladaLoader2.js

```diff
@@ -0,0 +1,3476 @@
+/**
+ * @author mrdoob / http://mrdoob.com/
+ * @author Mugen87 / https://github.com/Mugen87
+ *
+ *
+ * @Modified by Jihoon Lee from ColladerLoader.js@r88
+ * To support rviz compatible collada viewing.
+ * See: #202 why it is forked.
+ *
+ * It is a fork from ColladerLoader.js in three.js. It follows three.js license.
+ * Also it is renamed as ColladaLoader2.js
+ */
+
+THREE.ColladaLoader2 = function ( manager ) {
```

Comment:
Why did you name it ColladaLoader2? It's ColladaLoader, and we actually want it to replace THREE.ColladaLoader completely, don't we?

On a higher level, do we even still need to support two loaders? Looks like what was once ColladaLoader2 has replaced the original ColladaLoader completely by now anyway.

Reply:
Why do we need to replace it completely? It needs to be explicitly kept separate so that it indicates it is a modified version just for backward compatibility. Users should know that they use the modified version because their collada isn't following the standard.

Reply:
I disagree. 

My opinion is that users shouldn't have to care about what loader is used at all – de facto, we're not dealing with the Collada standard, but instead with the one set by RViz. From that perspective, all models that work in RViz are correct and we should display them the same way RViz does.

Either way, renaming the module to ColladaLoader2 is confusing because that's not what it is.  It has the exact same interface as ```THREE.ColladaLoader```.

Because of that, we could actually remove all the ```loader``` options from ```ros3djs``` and instead just use a different script tag to load either the official ColladaLoader version or this patched one:

```html
    <!-- Uncomment this whenever RViz starts handling up_axis
    <script src="http://static.robotwebtools.org/threejs/current/ColladaLoader.js"></script>
    -->
    <!-- Remove this when RViz starts handling up_axis -->
    <script src="http://static.robotwebtools.org/fixes/ColladaLoader.js"></script>
```

Reply:
Ok. reasonable. Also, I removed all loader option because it doesn't seem necessary.